src/pages/admin/CreateNewMeetingPage.vue

```diff
@@ -0,0 +1,118 @@
+<script lang="ts" setup>
+import { ref } from 'vue'
+import apis, { CreateMeetingRequest } from '@/lib/apis'
+import BaseInput from '@/components/UI/BaseInput.vue'
+import BaseButton from '@/components/UI/BaseButton.vue'
+import EmoineHeader from '@/components/EmoineHeader.vue'
+import { useRouter } from 'vue-router'
+
+const router = useRouter()
+
+//todo: 専用ヘッダー画像のアップロード
+const headerLogo = '/logo.png'
+
+const liveUrl = ref('')
+const isValidURL = ref(true)
+
+const getLiveIdFromUrl = (liveUrl: string) => {
+  const params = new URL(liveUrl).searchParams
+  const videoId = params.get('v')
+  if (!videoId) throw new Error('Invalid URL')
+  return videoId
+}
+
+const createMeeting = async () => {
+  let id = ''
+  try {
+    const videoId = getLiveIdFromUrl(liveUrl.value)
+    const meetingInfo: CreateMeetingRequest = {
+      videoId: videoId,
+      description: ''
+    }
+    const res = (await apis.createMeeting(meetingInfo)).data
+    id = res.id
+  } catch (e: unknown) {
+    if (e instanceof Error) {
+      console.log(e.message)
+      isValidURL.value = false
+      return
+    }
+  }
+
+  router.push({ name: 'admin/events/:eventId', params: { eventId: id } })
```

Comment:
多分ここが上手くページ遷移できてないんだけど、router.tsの設定によるから、router.tsに
```
 {
    path: '/admin/events/:eventId',
    name: 'AdminEventDetail',
    // eslint-disable-next-line @typescript-eslint/no-empty-function
    component: () => {} // TODO: ページ作成したらここに書く
  },
```
みたいなのを追加しておいてほしい

その上で、
```suggestion
  router.push({ name: 'AdminEventDetail', params: { eventId: id } })
```
みたいに、`name`のところはrouter.tsの`name`と一致させるようにすると追加ボタンを押したときに画面遷移してくれるはず(なんだけど、今はページまだ作ってないから遷移はしなくて:okk:動作確認したかったら他のと同じように適当なページをrouter.tsの`component`のところに設定してみるとよさそう)